dashboards/jetty/README.md

```diff
@@ -9,3 +9,8 @@
 |:------------------|
 |Filename: [jetty-gce-overview.json](jetty-gce-overview.json)|
 |This dashboard has charts displaying `Sessions`, `Session Time`, `Select Calls`, `Max Session Time`, `Threads`, and `Thread Queue` from Jetty as well as charts of infrastructure related metrics for the running Jetty VMs: `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`. There is also a card with links to docs and the Jetty logs in Cloud Logging.|
+
+|Jetty Prometheus Overview|
+|:------------------|
+|Filename: [jetty-prometheus-overview.json](jetty-prometheus-overview.json)|
+|This dashboard is based on prometheus metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including `Sessions`, `Session Time`, `Select Calls`, `Max Session Time`, `Threads`, and `Thread Queue`.|
```

Comment:
Update the exporter URL -- shouldn't be jmx_exporter?

Reply:
That's the exporter that was used to retrieve metrics.

Reply:
Ah my bad -- I was expecting the URL to be a specialized Jetty exporter and this was a copy-paste error, but I took a closer look at https://github.com/prometheus/jmx_exporter and figured out what was going on